charts/qfusion/v3.12.0/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: qfusion
+version: v3.12.0-rancher
+appVersion: v3.12.0-rancher
```

Comment:
Since `apiVersion: v2` has already been specified, please add `helmVersion: helm_v3` to deploy charts using the helm v3 version.

Reply:
```
apiVersion: v2
name: qfusion
version: v3.12.0-rancher
appVersion: v3.12.0-rancher
helmVersion: helm_v3
xxxx
```

Reply:
OK